docs/core/compatibility/serialization/5.0/deserializing-json-into-char-requires-single-character.md

```diff
@@ -28,21 +30,21 @@ JsonSerializer.Deserialize<char>("\"a\"");
 
 ## Reason for change
 
-<xref:System.Text.Json.JsonSerializer.Deserialize%60%601(System.String,System.Text.Json.JsonSerializerOptions)?displayProperty=nameWithType> parses text that represents a single JSON value into an instance of the type specified by the generic type parameter. Parsing should only succeed when the provided payload is valid for the specified generic type parameter. For a <xref:System.Char> value type, a valid payload is a single character string.
+Parsing for deserialization should only succeed when the provided payload is valid for the target type. For a `char` type, the only valid payload is a single-`char` string.
```

Comment:
```suggestion
Parsing for deserialization should only succeed when the provided payload is valid for the target type. For a <xref:System.Char> type, the only valid payload is a single-character string.
```

A suggestion to avoid being C#-centric. Consider applying throughout.

Reply:
"character" can't replace `char` because some characters are composed of multiple `chars`. I think using the xref link everywhere would result in too many links in the text.